RockWeb/Themes/NewSpring/Styles/_breakpoints.less

```diff
@@ -19,6 +19,11 @@
 	and (min-width: @screen-sm-min) 
 	and (max-width: @screen-sm-max) {
 
+	#cms-admin-footer {
```

Comment:
@briankalwat is this necessary with the class styling you declare later?

Reply:
@cdanwards the one later is an override specifically for mobile breakpoint.

Reply:
@briankalwat Ahh I see that now. sweet.